<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tomato_counterClass</class>
 <widget class="QMainWindow" name="tomato_counterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tomato_counter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="font">
         <font>
          <family>Microsoft YaHei Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>start work now</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="font">
         <font>
          <family>Microsoft YaHei Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="font">
         <font>
          <family>Microsoft YaHei Mono</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QLCDNumber" name="timeCounter">
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tomato_counter.qrc"/>
 </resources>
 <connections/>
</ui>
